src/data/history.ts

```diff
@@ -231,6 +249,8 @@ export const computeHistory = (
       unit = hass.config.unit_system.temperature;
     } else if (computeStateDomain(stateInfo[0]) === "humidifier") {
       unit = "%";
+    } else if (isNumerical(stateInfo)) {
+      unit = " ";
```

Comment:
Cant we move this check down, so we don't have to add a space as unit?

Reply:
It still needs a unit, because graphs are grouped by unit. But it could be `true`, I guess?

Reply:
Or maybe `#`?